packages/salesforcedx-vscode-apex-replay-debugger/src/commands/apexExecutionOverlayActionCommand.ts

```diff
@@ -46,12 +46,10 @@ export class ApexExecutionOverlayActionCommand extends BaseCommand {
     }
   }
 
-  // For this particular message the query string should be undefined
```

Comment:
For ApexExecutionOverlayActionCommand and BatchDeleteExistingOverlayActionCommand you deleted the comment about the query strings being undefined, why? That's a perfectly valid comment that describes part of the behavior of both commands.

Reply:
Because I was able to read the code which says you're passing in undefined to the parent constructor. That undefined argument is the query string. In fact, I think I'll remove `getQueryString()` all together because you're not doing anything different than baseCommand.ts